cpu-profiling/cl-position-handler/README.md

```diff
@@ -254,15 +254,33 @@ Refer to the following diagram showing the interaction diagram:
 ![prism-poc-prepare-and-position-handlers-in-isolation](assets/images/prism-poc-prepare-and-position-handlers-in-isolation.drawio.png)
 
 Before implementing prism, ran position handler and prepare handler as node services locally and dumped `10k` prepare kafka messages on `topic-transfer-prepare` and captured the readings as base line for this experiment.
+
+The following are the commands used to run preapre and position handlers as node processes without docker
+```
+export CLEDG_PORT=3001
+nohup node src/handlers/index.js handler --position 2>&1 > position.log &
+```
+
+```
+export CLEDG_PORT=3002
+nohup node src/handlers/index.js handler --prepare 2>&1 > prepare.log &
+```
+
 PRISM implementation can be found in this PR [central-ledger/pull/962](https://github.com/mojaloop/central-ledger/pull/962)
 The following is the comparison with PRISM PoC
 
-| Scenario | Cache    | Prepare Handler   | Position Handler  |
-|----------|----------|-------------------|-------------------|
-| Baseline | Disabled | 166 ops/s, 6.25ms | 138 ops/s, 7.62ms |
-| Baseline | Enabled  | 263 ops/s, 4.05ms | 151 ops/s, 7.76ms |
-| PRISM    | Disabled | 145 ops/s, 7.15ms | 141 ops/s, 7.44ms |
-| PRISM    | Enabled  | 228 ops/s, 4.52ms | 144 ops/s, 8.05ms |
+| Scenario | Cache    | Compression       | Prepare Handler   | Position Handler  |
+|----------|----------|-------------------|-------------------|-------------------|
```

Comment:
Please indicate what hardware was used for these results

Reply:
Done